src/barplot.ts

```diff
@@ -69,41 +78,61 @@ export class BarPlot extends AVisInstance implements IVisInstance {
     return <Element>this.$node.node();
   }
 
+  transform(scale?: [number, number], rotate: number = 0) {
+    const bak = {
+      scale: this.options.scale || [1, 1],
```

Comment:
` scale: this.options.scale.slice() || [1, 1],` gives me error:
Class 'BarPlot' incorrectly extends base class 'AVisInstance'.
  Types of property 'transform' are incompatible.
    Type '(scale?: [number, number], rotate?: number) => { scale: number[]; rotate: number; }' is not assignable to type '() => ITransform'.
      Type '{ scale: number[]; rotate: number; }' is not assignable to type 'ITransform'.
        Types of property 'scale' are incompatible.
          Type 'number[]' is not assignable to type '[number, number]'.
            Property '0' is missing in type 'number[]'.
 So does slice(0) ...